### PR TITLE
S4t2 create counter

### PR DESCRIPTION
**Pull request by gabrielrowan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,23 +22,29 @@ function enableEnterButton (length) {
 
 function outcomeOutput(bool, inputWord) {
     document.querySelector('.textInput').style.display = 'none'
-    document.querySelector('.submitFormButton').style.display = 'none'
+    /*document.querySelector('.submitFormButton').style.display = 'none'
+    document.querySelector('.retry').style.display = 'block'*/
```
Comment (Parsley-Afroman) [inline index.js:26]: please remove commented out code

File: styles.css [hunk 2]
```diff
@@ -240,16 +247,6 @@ form {
     display: flex;
 }
 
-/* toggling this class removes the enter button*/
-/*.enter{
-    display: none;
-}
```
Comment (Parsley-Afroman) [inline styles.css:246]: I would suggest removing the classes from the HTML document as they have become obsolete in this story; I would say that the same likely applies for the classes Input and Outcome that were used to present the changeability of the html element.

**Review by Parsley-Afroman — CHANGES_REQUESTED** [review]
Just a few things that require removal to make it tidier and I made a slight recommendation for centring the incorrect text output (would have been obvious to you i'm sure)

File: index.js (hunk 1 above)
Comment (gabrielrowan) [reply index.js:26]: Done

File: index.js [hunk 3]
```diff
@@ -59,6 +71,7 @@ fetch('words.json')
         window.location.reload()
     }
     document.cookie = "word=" + randomWord + ";expiry = Thu, 31 Dec 2037 12:00:00 UTC;"
+    console.log(randomWord)
```
[the remark this answers is not in the document]
Comment (gabrielrowan) [reply index.js:74]: Going to keep for now but we can always remove it before the final push at the end of the day

File: index.js [hunk 4]
```diff
@@ -88,16 +101,26 @@ allKeys.forEach((key) => {
...
+            matchResult = false
+        }
+        outcomeOutput(matchResult, guessedWord)
+    }
     if (guessedWord === randomWord) {
         matchResult = true
-    } else {
-        matchResult = false
+        outcomeOutput(matchResult, guessedWord)
+        console.log('bananas')
```
[the remark this answers is not in the document]
Comment (gabrielrowan) [reply index.js:119]: Removed :)

File: styles.css (hunk 2 above)
Comment (gabrielrowan) [reply styles.css:246]: Removed :)

**Review by Parsley-Afroman — COMMENTED** [review]
Happy with the explanation regarding keeping the console log

**Pull request merged**